build/GitCommitInfo.targets

```diff
@@ -27,7 +27,7 @@
 
     <WriteLinesToFile File="$(GitCommitInfoProps)"
                       Lines="$(GitCommitInfoPropsContent)"
-                      Condition=" '$(ShouldOverwriteThePropsFile)' == 'true' "
+                      Condition=" '$(ShouldOverWriteThePropsFile)' == 'true' "
```

Comment:
@eerhardt you're right of course but language is also constantly evolving - just my $0.02